### DISCUSSION TITLE
Displaying only the latest post and limiting downloaded feeds

### DISCUSSION THREAD

**Discussion by jakec-dev** [nivit/polybar-module-news#2] [Ideas]
Hi, is it possible to:

1. Display on the latest post

I'm not sure how the feeds are displayed by default - it seems to be totally random - but it would be fantastic if it were possible to only display the most recent post.

Use case: I have added various Github repo's to my feed so I can keep up to date with new releases. With the current implementation I have no idea whether the post being displayed is about a new release or old release that's in my news.items list and is being randomly displayed.

2. Limit downloaded feeds

I only want to keep posts that were published within the last 3 days with a maximum of the last 10 posts. Is there any way to do this? I don't need to keep hundreds of posts dating back to when I installed this module. It seems that `news.items` isn't cleared out at all so the list of posts just keeps building and building.

**Comment by nivit** [comment]
Hi,

at the moment all items are displayed sequentially as they appear in the feeds, but it should be easy to implement what you say.

Thanks for the suggestions!

**Comment by nivit** [comment]
Hi,

I have just implemented that feature in the [devel branch](https://github.com/nivit/polybar-module-news/tree/devel). You have to set up a value for the variable `max_news` in `$HOME/.config/scripts/news/conf/news.conf` after updating the module (see description in the repo).

HTH.
